JustSaying/JustSayingBus.cs

```diff
@@ -189,6 +189,16 @@ private IMessagePublisher GetActivePublisherForMessage(Message message)
             return publisher;
         }
 
+        private string GetActiveRegion()
+        {
+            if (Config.GetActiveRegion == null)
+            {
+                return Config.Regions.First();
+            }
+
+            return Config.GetActiveRegion();
```

Comment:
```csharp
=> Config.GetActiveRegion?.Invoke() ?? Config.Regions.First();
```

Reply:
Hm, I could go either way on that, does anyone want to break the tie on if the terse version is more readable?

Reply:
In this case I prefer the original, particularly if some handling needs adding to check there's something in the region set.